### ISSUE TITLE
General Issue for the Neos Discord

### ISSUE THREAD

**Issue by Holy-Person** [Neos-Metaverse/NeosPublic#3722]
### What topic do you want to discuss?

At the current moment the Neos Discord is a mess, even just fixing some of the minor issues mentioned can greatly improve the overall experience on the Discord.

### Goal of discussion

The current Neos Discord has a bunch of things that either don't make sense, are duplicates, not described correctly or just misused.
This issue is meant to be a place to collect different kinds of feedback/suggestions for the Discord.

### Discussion:

As an example, the "discord-groups' channel is mostly filled with old/expired invites.
I'd like to have the invites either renewed or removed, as just seeing "Invalid Invite" over and over again is kind of weird.
If the channel is completely unused and nothing is going to get renewed the whole channel might as well be archived.

**Comment by alizardguy** [comment]
With the influx of new users I've seen a lot of confusion about where users can have non-Neos related discussions, I think this is due in part to the #off-topic channel being in a non-immediate location whilst also being one of the only channels that dose not have an emoji or icon next to it. I feel the channel should possibly me moved to an easier to identify location and match it's categories naming convention (an icon prior the channel name).

**Comment by kdurmeter** [comment]
I personally like that discussions of other services get shunted to #off-topic.

The servers for other competing services is inundated with discussion about VRChat, and nothing to do with their respective services. It's all drowned out, and the fact that Neos seems to be focusing on showcasing what it has, over what other services do not is appreciated.

That being said, re-organizing the channels to make the #off-topic channel more prominent wouldn't be a bad thing.

In short, I'm personally happy with what is there. Though a re-organization does make sense.

Editing to add: I also appreciate the fact that moderators in the Discord server are on top of things. The other servers are currently overrun with casual racism and transphobia, which I have not seen on the Neos server.

**Comment by DoubleStyx** [comment]
Moving some stuff over that was previously mentioned:

There are some inconsistencies in the naming and sorting of the server emotes. The dash, locomotion, and inventory-button icons aren't grouped together properly. The capitalization of the mcc emotes is inconsistent. There are some other emotes that would benefit from grouping, such as the name badges and tooltips.

The #🗣neos-discussions and #🖥sites-and-news channels use colorless unicode symbols instead of emotes. Also, the names of the international channels may benefit from the addition of national flags.

**Comment by Holy-Person** [comment]
The #neos-discussions channel has the description of "Speak your mind, for cryptocurrency questions see: #neos-credits-ncr", tho according to multiple team members and moderators that channel is made for discussions that have to do with neos while the actual "Speak your mind" would be #off-topic. Maybe switch it to something like "Everything Neos" or "Talk about Neos".

**Comment by Holy-Person** [comment]
The #community-news channel is mostly just meetups by now and could be merged into #meetups by just relocating all the channel-follows/webhooks to #meetups instead. It also included projects from different communities before but most of them seem to no longer exist, or could be merged similar to the meetups but into channels like #user-creations.

**Comment by ProbablePrime** [comment]
Now the rush has died down, can we remove slow mode on the channels that had it?

or at least lower the slowmode time?

I assigned Raith(as he's the only one who i could for some reason), so he can forward the feedback along as it develops.
